src/types/platforms-types/sound.ts

```diff
@@ -1,13 +1,14 @@
 import _ from 'lodash';
 
 import { extractHashFromURL } from '../../clients/ipfs';
-import { ethereumId, slugify } from '../../utils/identifiers';
+import { evmId, slugify } from '../../utils/identifiers';
 import { ArtistProfile } from '../artist';
+import { ChainId } from '../chain';
 import { MapTrack, MapNFTsToTrackIds } from '../mapping';
 import { NFT, getNFTMetadataField } from '../nft';
 
-const mapAPITrackToArtistID = (apiTrack: any): string => {
-  return ethereumId(apiTrack.artist.user.publicAddress);
+const mapAPITrackToArtistID = (chainId: ChainId, apiTrack: any): string => {
+  return evmId(chainId, apiTrack.artist.user.publicAddress);
```

Comment:
should this be hardcoded to ethereum, since it's an artist ID? and done so using artistId() instead of evmId()?